### PR TITLE
chore: monthly release action

### PR DESCRIPTION
**Pull request by allisonsuarez**
### Summary of Changes

Added a workflow to create a new release every month.

### Tests

N/A

### Documentation

N/A

### CheckList

Make sure you have checked **all** steps below to ensure a timely review.

- [ ] PR title addresses the issue accurately and concisely. Example: "Updates the version of Flask to v1.0.2"
  - In case you are adding a dependency, check if the license complies with the [ASF 3rd Party License Policy](https://www.apache.org/legal/resolved.html#category-x).
- [ ] PR includes a summary of changes, including screenshots of any UI changes.
- [ ] PR adds unit tests, updates existing unit tests, **OR** documents why no test additions or modifications are needed.
- [ ] In case of new functionality, my PR adds documentation that describes how to use it.
  - All the public python functions and the classes in the PR contain docstrings that explain what it does
- [ ] PR passes all tests documented in the [developer guide](https://github.com/lyft/amundsenfrontendlibrary/blob/master/docs/developer_guide.md#testing)

### CODE REVIEW

**Review by Golodhros — APPROVED** [review]
LGTM

File: setup.cfg [hunk 1]
```diff
@@ -26,3 +26,9 @@ disallow_untyped_defs = True
 ignore_missing_imports = True
 strict_optional = True
 warn_no_return = True
+
+[semantic_release]
+version_variable = "amundsenfrontendlibrary/setup.py:__version__",
```
Comment (feng-tao) [inline setup.cfg:31]: nice, qq: how does the version get updated?

**Review by feng-tao — COMMENTED** [review]
could we do both auto release and manual release at the same time?

File: setup.cfg (hunk 1 above)
Comment (Golodhros) [reply setup.cfg:31]: the semantic-release library would do it for us.

**Comment by Golodhros** [comment]
> could we do both auto release and manual release at the same time?

Yeah, we were planning to support the manual release as current, by adding a tagged commit to the repo.

We also want to automatically create pre-releases with all merged PRs, still WIP

**Comment by mgorsk1** [comment]
does this take care of changelog generation as well or will be done in separate mr ?

**Pull request merged**